static/js/services/db-sync.js

```diff
@@ -120,12 +100,8 @@ var _ = require('underscore'),
         replicateDoneCallback(null, result);
       };
 
-      var startContinuousReplication = function(params) {
-        replicate('from', {
-          filter: 'erlang_filters/doc_by_place',
```

Comment:
Yes definitely. At this stage I just want to get something decent into develop so I can do a proper performance test. At which point I'll either clean up the filters and some other stuff, or revert this change.